v7/_posts/release notes/2018-06-26-ODL-7.5.0.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "ODataLib 7.5.0"
+description: "ODataLib 7.5.0 release notes"
+category: "4. Release Notes"
+---
+
+## Changes in ODataLib 7.5.0 Release ##
+
+## Notes ##
+
+ODataLib 7.5.0 includes the following items: IN operator, Entity set aggregation, various bug fixes, and performance improvement.
+
+## Features ##
+
+[[#757](https://github.com/OData/odata.net/pull/757)] Entity set aggregations
+
+[[#1165](https://github.com/OData/odata.net/pull/1165)] [Feature] IN operator
+
+
+## Fixed Bugs ##
+
+[[#513](https://github.com/OData/odata.net/issues/513)]	$select with complexCol/prop should be supported
```

Comment:
nit: there's a \t character between link and title

Reply:
fixed.

---
In reply to: [198338467](https://github.com/OData/odata.net/pull/1202#discussion_r198338467) [](ancestors = 198338467)

Reply:
ComplexCol is typo ?